acquisitions/src/main/resources/thunderjet/consortia/consortia-orders-junit.feature

```diff
@@ -59,9 +59,11 @@ Feature: mod-consortia integration tests
 
     # define main users
     * def consortiaAdmin = { id: '#(centralAdminId)', username: 'consortia_admin', password: 'consortia_admin_password', tenant: '#(centralTenant)'}
-    * def centralUser1 = { id: '#(centralUser1Id)', username: 'central_user1', password: 'central_user1_password', type: 'staff', tenant: '#(centralTenant)'}
     * def universityUser1 = { id: '#(universityUser1Id)', username: 'university_user1', password: 'university_user1_password', type: 'staff', tenant: '#(universityTenant)'}
 
+    * def centralUser1 = { id: '#(centralUser1Id)', username: 'central_user1', password: 'central_user1_password', type: 'staff', tenant: '#(centralTenant)'}
+    * def centralUser1Perms = { id: '#(centralUser1Id)', permissions: ['orders.all'], tenant: '#(centralTenant)'}
```

Comment:
we don't have pattert to set 'orders.all' for this test, do we really need it?

Reply:
I copied user permissions from mod-orders module and assign it here